src/garage/tf/plotter/plotter.py

```diff
@@ -1,3 +1,4 @@
+"""Renders rollouts of the policy as it trains."""
```

Comment:
i have a mental TODO to move this and garage.plotter into garare.experiment, but does not need to be done here.